string type
float32 damage
float32 fire_rate
float32 accuracy
float32 reload_time
